### PR TITLE
Fixes2

### PR DESCRIPTION
**Pull request by beingsebi**

### CODE REVIEW

File: backtester/views.py [hunk 1]
```diff
@@ -155,6 +154,7 @@ def form_valid(self, form):
 
         response = super().form_valid(form)
         # Start the expensive task in the background
+        # update_results(self.object.pk)
```
Comment (VoicilaIonut) [inline backtester/views.py:157]: please remove

File: backtester/views.py [hunk 2]
```diff
@@ -196,4 +196,6 @@ def form_valid(self, form):
         response = super().form_valid(form)
         # Start the expensive task in the background
         expensive_task.delay(self.object.pk)
+        # update_results(self.object.pk)
```
Comment (VoicilaIonut) [inline backtester/views.py:199]: please remove

File: environment.yml [hunk 3]
```diff
@@ -1,9 +1,9 @@
-name: alphatester
+name: alphatester2
```
Comment (VoicilaIonut) [inline environment.yml]: change to alphatester

File: utils/database/strat_runner_results_to_db.py [hunk 4]
```diff
@@ -11,15 +11,18 @@
...
-    # print(results)
-    strategy.results = Results.toJSON(results)
-    # todo might have to call json pickle here
-    # strategy.description = "This is a test description"
+        transactions, strat.capitalAllocation,
+        strat.startDatetime)  # interpret the transactions and get the results
+
+    strategy.results = Results.toJSON(
+        results)  # convert the results to a JSON string
+
```
Comment (VoicilaIonut) [inline utils/database/strat_runner_results_to_db.py]: please add comments before the code for better formatting

File: utils/strategy/amount.py [hunk 5]
```diff
@@ -1,4 +1,4 @@
-class Amount:
+class Amount:  # either fixed or percentage
```
Comment (VoicilaIonut) [inline utils/strategy/amount.py:1]: same here

**Review by VoicilaIonut — CHANGES_REQUESTED** [review]

**Comment by beingsebi** [comment]
solved requested changes

**Review by VoicilaIonut — APPROVED** [review]

**Pull request merged**